pyface/i_heading_text.py

```diff
@@ -60,9 +60,14 @@ def __init__(self, parent=None, **traits):
             self.create()
             warnings.warn(
                 "automatic widget creation is deprecated and will be removed "
-                "in a future Pyface version, use create=False and explicitly "
-                "call create() for future behaviour",
-                PendingDeprecationWarning,
+                "in a future Pyface version, code should not pass the create "
+                "parameter and should instead call create() explicitly",
+                DeprecationWarning,
```

Comment:
It may be worth adding a suitable `stacklevel` parameter here so that the warning message points to the client code (i.e., the code that's passing the `create` parameter).

Also applies to other uses of `DeprecationWarning` in this PR.